src/v2/port_mapping.in.rs

```diff
@@ -58,6 +58,42 @@ impl FromStr for Ports {
     }
 }
 
+/// An IP protocol
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum Protocol {
+    /// Transmission Control Protocol, the default.
+    TCP,
+    /// User Datagram Protocol.
+    UDP
```

Comment:
Just an incredible minor style nit. :-) Rust seems to prefer writing acronyms in APIs as `Tcp` and `Udp` most of the time.

Reply:
👍